server/controllers/temporary-accommodation/manage/assessmentsController.test.ts

```diff
@@ -254,10 +254,10 @@ describe('AssessmentsController', () => {
           })
         })
 
-        describe('when there is a CRN search error', () => {
+        describe('when there is a search error', () => {
```

Comment:
I believe we're not showing errors, instead clearing the search when there's no search term? This test name should reflect that.

Reply:
Or be deleted altogether!

Reply:
I don't think this is relevant anymore